src/contracts/atlas/ExecutionEnvironment.sol

```diff
@@ -133,7 +134,7 @@ contract ExecutionEnvironment is Base {
     function postOpsWrapper(bytes calldata returnData)
         external
         onlyAtlasEnvironment(ExecutionPhase.PostOps, _ENVIRONMENT_DEPTH)
-        contributeSurplus(Party.DApp)
+        contributeSurplus
```

Comment:
Should this be `payable`? It's receiving value from the EE call.

Reply:
Yes, added